external-crates/move/crates/move-command-line-common/src/parser.rs

```diff
@@ -189,13 +210,12 @@ impl<'a, I: Iterator<Item = (TypeToken, &'a str)>> Parser<'a, TypeToken, I> {
                     _ => vec![],
                 };
                 ParsedType::Struct(ParsedStructType {
-                    address,
-                    module: module_contents.to_owned(),
-                    name: struct_contents.to_owned(),
+                    module,
+                    name,
```

Comment:
clarification: having already parsed the module and type, we can construct ParsedStructType from them

Reply:
Yes, this hasn't really changed, I just did this to make sure there's only one place where parsing of `ModuleId` is implemented, so now we treat a parsed struct type as a ModuleId, followed by `::[name]<[type-param],*>`.